addons/my-ocular/profile.js

```diff
@@ -0,0 +1,42 @@
+export default async function ({ addon, global, console }) {
+  var username = document.querySelector("#profile-data > div.box-head > div > h2").innerText;
+
+  var container = document.querySelector(".location");
+
+  var response = await fetch(`https://my-ocular.jeffalo.net/api/user/${username}`);
+  var data = await response.json();
+
+  var statusText = data.status;
+  var color = data.color;
+  if (statusText) {
+    var statusSpan = document.createElement("i"); // for whatever reason, chrome turns variable named status into text. why the heck. aaaaaaaaaaaaaaaaaa
+    statusSpan.title = `This is ${username}'s ocular status, displayed with Scratch Addons. You can set one at https://my-ocular.jeffalo.net.`;
+    statusSpan.innerText = statusText;
+
+    var dot = document.createElement("span");
+    dot.title = `This is ${username}'s ocular favourite colour, displayed with Scratch Addons. You can set one at https://my-ocular.jeffalo.net.`;
```

Comment:
This will require localization. I suggest you also remove the username placeholder for easier translation, changing it to "this user's" or something like that.

Reply:
I'd also merge both strings (favorite color and ocular status) into a single one.

Reply:
👍